source/fundamentals/crud.txt

```diff
@@ -1,13 +1,26 @@
-
-====
-CRUD
-====
+===============
+CRUD Operations
+===============
 
 .. default-domain:: mongodb
 
 .. toctree::
-   :titlesonly:
-   :maxdepth: 1
+   :caption: CRUD Operations
+
+   /fundamentals/crud/read-operations
+   /fundamentals/crud/write-operations
+
+CRUD (Create, Read, Update, Delete) operations allow you to work with
+the data stored in MongoDB.
+
+The CRUD operation documentation is categorized in two sections:
+
+- :doc:`Read Operations </fundamentals/crud/read-operations>` find and return
+  documents stored within your MongoDB database.
+- :doc:`Write Operations </fundamentals/crud/write-operations>` insert, modify,
+  or delete documents in your MongoDB database.
 
-   /fundamentals/crud/read
-   
\ No newline at end of file
+..
+  Some operations combine aspects of read and write operations. See our
+  guide on :doc:`compound operations </fundamentals/crud/compound-operations>`
+  to learn more about these hybrid methods.
```

Comment:
Question: is hybrid the term we use for compound methods? It seems like terminology to me, so perhaps we should avoid it if it's not the official term and we're not explaining what makes them hybrid.

Reply:
I'm not aware of any approved term. This is commented out and will only be included in once we have a page or section for compound operations, and I agree that at that time we should look at making this more approachable with as few "jargon" words as possible.